src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal, Button, withBreakpoints } from 'cozy-ui/react'
+import { Query } from 'cozy-client'
+import Topbar from 'drive/web/modules/layout/Topbar'
+import { ROOT_DIR_ID, TRASH_DIR_ID } from 'drive/constants/config'
+import Alerter from 'cozy-ui/react/Alerter'
+
+import cx from 'classnames'
+
+import FileList from 'drive/web/modules/filelist/FileList'
+import FileListHeader, {
+  MobileFileListHeader
+} from 'drive/web/modules/filelist/FileListHeader'
+import FileListBody from 'drive/web/modules/filelist/FileListBody'
+import FileListRows from 'drive/web/modules/filelist/FileListRows'
+import File from 'drive/web/modules/filelist/File'
+import fileListStyles from 'drive/styles/filelist'
+
+import {
+  Breadcrumb,
+  PreviousButton,
+  renamePathNames
+} from 'drive/web/modules/navigation/Breadcrumb'
+import getFolderPath from 'drive/web/modules/navigation/getFolderPath'
+
+const MoveTopbar = withBreakpoints()(
+  ({ navigateTo, path, breakpoints: { isMobile } }) => (
+    <Topbar hideOnMobile={false}>
+      {path.length > 1 &&
+        isMobile && (
+          <PreviousButton onClick={() => navigateTo(path[path.length - 2])} />
+        )}
+      <Breadcrumb path={path} onBreadcrumbClick={navigateTo} opening={false} />
+      <div role="toolbar">add folder</div>
+    </Topbar>
+  )
+)
```

Comment:
Can this component have its specific file ? ;)

Reply:
Absolutely yes. I'm planning on splitting things up further and into more files when I'm closer to finishing the PR.